index.html

```diff
@@ -100,6 +100,48 @@ <h1 class="display-3 text-center" id="directorHeading">D I R E C T O R</h1>
       </div>
     </div>
 
+    <div class="container">
+      <div class="row mt-5">
+        <div class="col-12 text-center">
+          <p>As a trusted partner, Jetpharma offers</p>
+          <h1 class="display-3" id="servicesHeading">S E R V I C E S</h1>
+        </div>
+      </div>
+      <div class="row" id="servicesTextRow">
+        <div class="col-lg-4 px-5">
+          <img src="assets/regulatory-affairs-icon.svg" alt="Regulatory Affairs Icon" class="servicesIconImages rounded mx-auto d-block">
+          <br />
+          <p class="text-dark text-center font-weight-bold">Regulatory Affairs</p>
+          <p>- Our team has a proven track record of approvals for pharmaceutical, OTC, traditional products and medical devices.</p>
+          <br />
+        </div>
+        <div class="col-lg-4 px-5">
+          <img src="assets/salesforce-marketing-icon.svg" alt="Regulatory Affairs Icon" class="servicesIconImages rounded mx-auto d-block">
+          <br />
+          <p class="text-dark text-center font-weight-bold">Salesforce & Marketing</p>
+          <p>
+            - Effective salesforce with a wide coverage of various channels from private hospitals and clinics to pharmacies <br />
+            - Experienced management team armed with the ability to draw up marketing strategies to build value and grow products across various therapeutic classes
+          </p>
+          <br />
+        </div>
+        <div class="col-lg-4 px-5">
+          <img src="assets/warehousing-facilities-icon.svg" alt="Regulatory Affairs Icon" class="servicesIconImages rounded mx-auto d-block">
+          <br />
+          <p class="text-dark text-center font-weight-bold">Warehousing Facilities & Distribution</p>
+          <p>
+              -	With a new expanded warehousing facility which adheres to Good Distribution Practices for Medical Devices <br />
+              -	A wide distribution network which allows us to reach customers in every corner of the country promptly               
+          </p>
+          <br />
+        </div>
+      </div>
+        
```

Comment:
remove this empty line. closing element tag shouldn't have empty line above them for readility